app/ide-desktop/lib/dashboard/src/authentication/src/authentication/cognito.ts

```diff
@@ -27,6 +27,11 @@ const MESSAGES = {
         userNotConfirmed: 'User is not confirmed. Please check your email for a confirmation link.',
         incorrectUsernameOrPassword: 'Incorrect username or password.',
     },
+    forgotPassword: {
+        userNotFound: 'User not found. Please register first.',
+        userNotConfirmed:
+            'Cannot reset password for user with unverified email. Please verify your email first.',
```

Comment:
not correct English sentence in user-facing messaging. Missing "an". Even better, you can ask ChatGPT to reformat it nicer :)